src/app.js

```diff
@@ -1 +1,36 @@
+/* eslint-disable no-console */
 // write code here
+const fs = require('fs/promises');
+const path = require('path');
+
+async function moveFile() {
+  const [source, destination] = process.argv.slice(2);
+
+  if (!source || !destination) {
+    console.log(!source);
```

Comment:
The console.log statement here seems unnecessary and might be a leftover from debugging. Consider removing it to clean up the output.